app/Http/Controllers/Contacts/NotesController.php

```diff
@@ -15,13 +17,17 @@ class NotesController extends Controller
      */
     public function get(Contact $contact)
     {
+        $client = CouchDbHelper::getAccountDatabase($contact->account_id);
+
         $notesCollection = collect([]);
-        $notes = $contact->notes()->latest()->get();
+        $response = $client->limit(7)->startkey([$contact->id, []])->endkey([$contact->id])->descending(true)->include_docs(true)->getView('notes', 'byContact');
```

Comment:
Could we use a method for this? It looks so complicated, and if we have to enter it again and again...

Reply:
Right.